packages/contracts/test/LibExperience.t.sol

```diff
@@ -27,4 +33,56 @@ contract LibExperienceTest is MudV2Test {
       assertEq(exp[i], 1);
     }
   }
-}
\ No newline at end of file
+
+  function testGetExp() public {
+    uint32[PStat_length] memory exp = [uint32(10), 20, 30];
+    Experience.set(targetEntity, exp);
+
+    uint32[PStat_length] memory retrievedExp = LibExperience.getExp(targetEntity);
+    for (uint256 i; i < retrievedExp.length; i++) {
+      assertEq(retrievedExp[i], exp[i]);
+    }
+  }
+
+  function testAddExp() public {
+    LibExperience.initExp(targetEntity);
+
+    uint32[PStat_length] memory initialExp = LibExperience.getExp(targetEntity);
+
+    uint32[PStat_length] memory addExp = [uint32(5), 10, 15];
+    LibExperience.increaseExp(targetEntity, addExp);
+
+    uint32[PStat_length] memory expectedExp;
+    for (uint256 i; i < initialExp.length; i++) {
+      expectedExp[i] = initialExp[i] + addExp[i];
+    }
+
+    uint32[PStat_length] memory retrievedExp = LibExperience.getExp(targetEntity);
+    for (uint256 i; i < retrievedExp.length; i++) {
+      assertEq(retrievedExp[i], expectedExp[i]);
+    }
+  }
```

Comment:
this is still just like `testIncreaseExp`, but looks needlessly complicated and with slightly different values